.github/workflows/rag-tests.yml

```
@@ -0,0 +1,45 @@
+          ollama pull llama2
+          
+      - name: Run Ollama RAG Tests
+        env:
+          KANBN_ENV: test
+          DEBUG: true
+          KANBN_TEST_MODE: true
+          OLLAMA_HOST: http://localhost:11434
+          OLLAMA_MODEL: llama2
+          USE_OLLAMA: true
+        run: bash ./examples/test_ollama_rag_integrations\ copy.sh
+        
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)